app/routes.js

```diff
@@ -15,7 +15,7 @@ export default (
     <Route path="/app" component={App}>
       <Redirect from="/" to="/app"/>
       <IndexRoute component={HomePage}/>
-      <Route path="/app/users" component={UsersPage}/>
+      <Route path="/app/users" component={UsersPageComponent}/>
```

Comment:
I agree with this change, but maybe we should do all of the imports together instead of just one. In that way, we could avoid creating inconsistencies.

Reply:
Replied on the other PR. Thanks